example_test.go

```diff
@@ -53,13 +53,22 @@ func ExampleUnmarshal() {
 	result, err = marshmallow.Unmarshal([]byte(`{"foo":2,"boo":[1,2,3]}`), &v,
 		marshmallow.WithMode(marshmallow.ModeFailOverToOriginalValue))
 	fmt.Printf("ModeFailOverToOriginalValue and invalid value: result=%+v, err=%T\n", result, err)
+
+	// unmarshal with mode marshmallow.ModeExcludeKnownFieldsFromReturnedMap and valid value
+	// this will return unmarshalled result excludes the typed fields
```

Comment:
what do you think about "unmarshalled result that contains unknown fields only"
or  "unmarshalled result without known fields"